src/middlewared/middlewared/service/core_service.py

```diff
@@ -593,6 +650,7 @@ async def download(self, app, method, args, filename, buffered):
         return job.id, f'/_download/{job.id}?auth_token={token}'
 
     @private
+    @no_authz_required
```

Comment:
This is a private method for testing, this decorator is not necessary there

Reply:
I was removing authorization requirement so that I could use it for testing job behavior with various limited users.